Rakefile

```diff
@@ -50,17 +50,17 @@ namespace :crowbar do
     ).flatten.compact.uniq
   end
 
-  desc "Init all barclamps"
+  desc "Initialize all barclamps (:fork, :clone, :add_upstream, :add_susecloud)"
```

Comment:
@tboerger I agree, although it's better than nothing.  But maybe you have a better suggestion?